doc/user/ruby-managers.md

```diff
@@ -81,6 +81,14 @@ You can also install the TruffleRuby JVM Standalone with:
 asdf install ruby truffleruby+graalvm-VERSION OR truffleruby+graalvm-dev
 ```
 
+## `mise` (with ruby mise core plugin)
+
+You can install a TruffleRuby Native Standalone release or dev build with:
+
+```bash
+mise install ruby@truffleruby-VERSION OR truffleruby-dev
+```
```

Comment:
I think `truffleruby+graalvm-VERSION OR truffleruby+graalvm-dev` should also work, could you try it and document it there?

Reply:
> mise install ruby@truffleruby+graalvm-24.2.0

works fine, too

Reply:
Thanks for checking, could you also document that, using the same structure as for asdf just above?

Reply:
@eregon, sure, let me know if it's ok now.

Reply:
Yes, that's great, thank you.